nuxeo-core/nuxeo-core-lock/src/main/java/org/nuxeo/ecm/core/lock/LockHelper.java

```diff
@@ -27,8 +27,6 @@
 import org.nuxeo.runtime.kv.KeyValueStore;
 import org.nuxeo.runtime.transaction.TransactionHelper;
 
-import java.util.concurrent.Callable;
-import java.util.function.Function;
 import java.util.function.Supplier;
```

Comment:
Could you also move `LockHelper` to `org.nuxeo.ecm.core.api.LockHelper` in `nuxeo-core-api` and remove `nuxeo-core-lock` to avoid having a bundle just for one helper class?